jipstatus.py

```diff
@@ -29,16 +29,23 @@ def add_domain(user):
         user = user + "@linaro.org"
     return user
 
-def enumerate_updates(jira):
+def default_jql():
     user = cfg.args.user
     project = cfg.args.project
 
+    if project:
+        jql = "project =  '%s' " % project
+        jql += "AND assignee = '%s' " % add_domain(user)
```

Comment:
that looks wrong. I tried that locally, and if I use --project QLT it only gives me my cards, not the one from the whole project. 

so perhaps we should add user only if project and team are not set? does that make sense? running without args , means it should return 'my' cards. running with --user xx it should return cards for xx, and using --project or/and --teams overrides --user?

Reply:
Btw, just want to mention that I'm following a long with your discussion. But I'll waiting giving it a try again until you guys are in agreement. The only comment I have is that, have in mind that target user here might be another team (or another user). But it seems like you're working on fixing that, thanks!

Reply:
@jbech-linaro , i reviewed all patches, and tested it here. Looks good for me!
thanks a lot @robertfoss !